doc/topics/releases/releasecandidate.rst

```diff
@@ -43,12 +43,14 @@ Salt version and release:
 
 .. code-block:: bash
 
+    sudo rpm --import https://repo.saltproject.io/salt/py3/redhat/9/x86_64/minor/<salt version>/SALTSTACK-GPG-KEY2.pub
```

Comment:
This implies SALTSTACK-GPG-KEY2.pub for all redhat, how about something along the lines:

if $releasever == 9 then
sudo rpm --import https://repo.saltproject.io/salt/py3/redhat/9/x86_64/minor/<salt version>/SALTSTACK-GPG-KEY2.pub
baseurl=https://repo.saltproject.io/salt_rc/salt/py3/redhat/9/$basearch/minor/<salt version and release number>
else:
 sudo rpm --import https://repo.saltproject.io/salt/py3/redhat/$releasever/x86_64/minor/<salt version>/SALTSTACK-GPG-KEY.pub
baseurl=https://repo.saltproject.io/salt_rc/salt/py3/redhat/$releasever/$basearch/minor/<salt version and release number>
fi

Not sure if pseudo code has to correct